android/source/VideoLocker/src/org/edx/mobile/view/FindCoursesActivity.java

```diff
@@ -0,0 +1,65 @@
+package org.edx.mobile.view;
+
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import org.edx.mobile.R;
+import org.edx.mobile.base.BaseFragmentActivity;
+import org.edx.mobile.util.AppConstants;
+import org.edx.mobile.util.NetworkUtil;
+
+public class FindCoursesActivity extends BaseFragmentActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_find_courses);
+
+        // configure slider layout. This should be called only once and
+        // hence is shifted to onCreate() function
+        configureDrawer();
+
+        try{
+            segIO.screenViewsTracking(getString(R.string.find_courses_title));
+        }catch(Exception e){
+            logger.error(e);
+        }
+
+        if (!(NetworkUtil.isConnected(this))) {
+            AppConstants.offline_flag = true;
+        }else{
+            AppConstants.offline_flag = false;
+        }
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+        setTitle(getString(R.string.find_courses_title));
+    }
+
+    @Override
+    protected void onOffline() {
+        AppConstants.offline_flag = true;
+    }
+
+    @Override
+    protected void onOnline() {
+        AppConstants.offline_flag = false;
```

Comment:
Yes, the parent already does this, but had added it as we will have to implement offline mode UI but as the offline mode UI was not finalized did not remove it.
I will remove this for this PR and will reintroduce this if required when the offline mode UI is finalized.